app/javascript/src/Types/swagger.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'swagger-client/es/execute' {
+  export function execute (req: unknown): unknown
+}
```

Comment:
Why is this required and why is it under the `Types` directory?

Reply:
Well, that's basically types and modules definitions... Typescript requires this.

Reply:
But it's not this method already defined in the library?